Backendless/Persistence/DataQueryBuilder.cs

```diff
@@ -122,6 +122,15 @@ public DataQueryBuilder SetRelationsDepth( int relationsDepth )
       return queryOptionsBuilder.SetRelationsDepth( relationsDepth );
     }
 
+    public int GetRelationsPageSize()
+    {
+        return queryOptionsBuilder.GetRelationsPageSize();

```

Comment:
indent >> 2 spaces (make sure to check it everywhere). If needed adjust code formatting rules in VS